lib/namespace.js

```diff
@@ -62,14 +63,14 @@ function Namespace(server, name){
  * Inherits from `EventEmitter`.
  */
 
-Namespace.prototype.__proto__ = Emitter.prototype;
+util.inherits(Namespace, Emitter);
```

Comment:
Modified inheritance to use node.js `util`. This is because ESLint throws an error, mentioning that the `__proto__` way is deprecated.